libraries/rtos/rtx/TARGET_ARM7/RTX_Conf_CM.c

```diff
@@ -204,17 +204,12 @@
 /*----------------------------------------------------------------------------
  *      OS Idle daemon
  *---------------------------------------------------------------------------*/
+extern void rtos_idle_loop(void);
```

Comment:
I was mirroring the way `mbed_die()` is forward declared for the error functions, and forgot to remove the unnecessary declaration from _rtos_idle.h_. Fixed in c52e0dc.